src/null-safety/unsound-null-safety.md

```diff
@@ -0,0 +1,178 @@
+---
+title: Unsound null safety
+description: Mixing language versions lets you migrate to null safety at your own pace, with some of the benefits of null safety.
+---
+
+An app can use some libraries that
```

Comment:
Packages can do this too. Given we're focusing on package migration for beta, perhaps clarify that here.